src/dynamic_tab.rs

```diff
@@ -43,6 +44,11 @@ pub trait Tab {
     fn inner_margin(&self) -> Margin {
         Margin::same(4.0)
     }
+
+    /// Whether the tab will be cleared with the color specified in [Style::tab_background_color`](crate::Style::tab_background_color)
```

Comment:
typo 
```suggestion
    /// Whether the tab will be cleared with the color specified in [Style::tab_background_color](crate::Style::tab_background_color)
```
![typo](https://user-images.githubusercontent.com/53652728/189522200-19f65818-f987-4896-80c6-0affa3c9ae77.png)